### PR TITLE
Add GitHub templates 

### PR DESCRIPTION
**Pull request by dave-v0**
**Preview Link:**

https://pr-2-sef-handbook.surge.sh/

### CODE REVIEW

**Comment by dave-v0** [comment]
@anjula-sack  Added the GitHub templates

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,34 @@
...
+
+### Screenshots
+<!---  Include an animated GIF or screenshot if the change affects the UI.  -->
+  
+### Preview Link
 <!---  This PR will be automatically deployed to surge. -->
-<!---  Once you submit the PR, replace "{PR_NUMBER}" with your PR number. ex: https://pr-23-sef-site.handbook.sh/ -->
-https://pr-{PR_NUMBER}-sef-handbook.surge.sh/
+<!---  Once you submit the PR, replace "{PR_NUMBER}" with your PR number. -->
+https://pr-{PR_NUMBER}-sef-site.surge.sh/
```
Comment (anjula-sack) [inline .github/PULL_REQUEST_TEMPLATE.md]: This part is not required. @DilinaBandara

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Comment by anjula-sack** [comment]
Can you link the issue to the description?

**Review by anjula-sack — APPROVED** [review]
Thank you! @DilinaBandara

**Pull request merged**